### ISSUE TITLE
Can't search for profiles until logged-in

### ISSUE THREAD

**Issue by BenDoernberg** [onenameio/support#3]

**Comment by muneeb-ali** [comment]
I've also noticed it. This is not linked to being logged in though, the search bar shows up on user profiles and a few other views but not all views e.g.,

https://onename.com/muneeb  <--- the search works there even if you're not logged in

**Comment by shea256** [comment]
The search used to be there on the landing page. It was taken away so to direct people to the desired action / so newcomers aren't confused.

**Comment by shea256** [comment]
Maybe the benefits of adding it outweigh the downsides, but I was thinking that when you're on the landing page and in the signup flow, you shouldn't be distracted and get diverted away from what we want you to do - signup.

**Comment by BenDoernberg** [comment]
I do think requiring a login to use the "phonebook" is an annoyance for existing users, *especially on mobile*. Would it be possible to A/B test to find out how big a distraction a search bar would be for new user signups?

**Comment by shea256** [comment]
Yeah, actually on mobile, search doesn't even show up at all right now. I need to design a different search view for mobile (the desktop one just didn't display even close to right).
